moped-editor/src/components/GridTable/Filters.js

```diff
@@ -685,13 +699,15 @@ const Filters = ({
                       />
                     ) : (
                       <TextField
+                        error={isInvalidInput}
+                        helperText={isInvalidInput ? "Invalid input" : ""}
```

Comment:
is this helpful enough or should it be more specific "Invalid input, numbers only" or something

Reply:
It'd be good to be more specific. maybe "Must be a number" ?

Reply:
Just adding that it makes me so happy to see feedback here 🙌